src/app/public/modules/phone-field/phone-field.component.spec.ts

```diff
@@ -0,0 +1,849 @@
+import {
+  ComponentFixture,
+  TestBed,
+  fakeAsync,
+  tick
+} from '@angular/core/testing';
+
+import {
+  FormsModule,
+  NgModel,
+  ReactiveFormsModule
+} from '@angular/forms';
+
+import {
+  By
+} from '@angular/platform-browser';
+
+import {
+  expect
+} from '@skyux-sdk/testing';
+
+import {
+  PhoneFieldTestComponent
+} from './fixtures/phone-field.component.fixture';
+
+import {
+  PhoneFieldReactiveTestComponent
+} from './fixtures/phone-field-reactive.component.fixture';
+
+import {
+  SkyPhoneFieldModule
+} from './phone-field.module';
+
+describe('Phone Field Component', () => {
+
+  function setInput(
+    element: HTMLElement,
+    text: string,
+    compFixture: ComponentFixture<any>) {
+    let inputEvent = document.createEvent('Event');
+    let params = {
+      bubbles: false,
+      cancelable: false
+    };
+    inputEvent.initEvent('input', params.bubbles, params.cancelable);
+
+    let changeEvent = document.createEvent('Event');
+    changeEvent.initEvent('change', params.bubbles, params.cancelable);
```

Comment:
Possible to use the `SkyAppTestUtility.fireDomEvent` instead?

Reply:
Yep. Updated.